src/frontend/src/views/Upload/components/Review/components/Upload/style.ts

```diff
@@ -43,3 +44,22 @@ export const StyledDialogActions = styled(DialogActions)`
   ${centerContent}
 `;
 
+export const StyledUploadImage = styled(UploadImage)`
+  ${(props) => {
+    const colors = getColors(props);
+
+    return `
+      fill: ${colors?.primary[400]};
+    `;
+  }}
+`;
+
+export const StyledUploadFailedImage = styled(UploadFailedImage)`
+  ${(props) => {
+    const colors = getColors(props);
+
+    return `
+      fill: ${colors?.error[400]};
```

Comment:
oh cool you got the colors working without modifying the svg directly! How did you fix that? Or was i just doing something wrong? lol

Reply:
Ah yes, I removed `fill` from SVG file, so we can overwrite it through our styled components 🎉 !